osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -208,6 +208,14 @@ protected override void LoadComplete()
             customisationVisible.BindValueChanged(_ => updateCustomisationVisualState(), true);
 
             updateAvailableMods();
+
+            // Start scrolled slightly to the right to give the user a sense that
+            // there is more horizontal content available.
+            ScheduleAfterChildren(() =>
+            {
+                columnScroll.ScrollTo(100, false);
```

Comment:
100 feels quite small to notice, and I understand that going higher or using `DisplayableContent`/`ScrollableExtent` would break the show transition completely, but 200 may be better at least?

- 100:

  https://user-images.githubusercontent.com/22781491/167292171-9e351425-3a30-45df-8b37-ad7d9756b813.mp4

- 200:

  https://user-images.githubusercontent.com/22781491/167292160-1555bca9-9e4d-4b5a-9284-287fb4d4dee2.mp4

Reply:
Sure, I almost chose 300 so that might be a good middle-ground.